src/statsmodel.jl

```diff
@@ -180,6 +180,21 @@ function StatsBase.coeftable(model::TableModels; kwargs...)
     ct
 end
 
+_show_fit_stats(io::IO, model::TableModels) = nothing
+
+function _show_fit_stats(io::IO, model::TableRegressionModel)
+    println("R²: ", round(r2(model), sigdigits=4),
+         "\t Adjusted R²: ", round(adjr2(model), sigdigits=4))
```

Comment:
This should also be included in a `try... catch` block, right? For GLMs (or any model not implementing `r2`) this will fail.

Reply:
I agree, changed it. Thank you.